x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -188,7 +189,7 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({
           chartSourceData = fillMissingChartData(chartSourceData, chartBucketData);
         }
       }
-
+      setAllEmptyValues(!chartBucketData.some((datum) => datum[1] !== null && datum[1] !== 0));
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/elastic/kibana/commit/44210b03c3e084648fc06068d7effd5c32e35de6